### PR TITLE
housekeeping: Added ReactiveUI.Splat.Tests

### PR DESCRIPTION
**Pull request by RLittlesII**
**What kind of change does this PR introduce?**

Added Splat Adapter tests to wrap up https://github.com/reactiveui/splat/issues/287

**What is the current behavior?**

There is no test coverage.

**What is the new behavior?**

ReactiveUI platform registration is tested against Splat adapters

**What might this PR break?**

Nothing

**Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Splat.Autofac" Version="7.*" />
+    <PackageReference Include="Splat.DryIoc" Version="7.*" />
+    <PackageReference Include="Splat.Ninject" Version="7.*" />
+    <PackageReference Include="xunit" Version="2.4.0" />
```
Comment (glennawatson) [inline src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj]: Remove these, they are old and the directory.build.props files will get these.

Comment (glennawatson) [reply src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj]: The xunit is what i'm referring to btw.

Comment (RLittlesII) [reply src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj]: removed.

**Comment by glennawatson** [comment]
In case anyone is curious why the code coverage stats didn't change it's because we only do code coverage on DLLs with reactiveui in the name. These tests don't add any extra coverage to ReactiveUI but are tests originally in the splat project and are very coupled with Reactiveui directly. We don't want to add splat to the coverage otherwise we'd have to cover all of splat in our unit tests.

**Pull request merged**